Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -649,6 +649,15 @@ void MainWindow::ConnectHotkeys()
     movie.SetReadOnly(read_only);
     emit ReadOnlyModeChanged(read_only);
   });
+#ifdef USE_RETRO_ACHIEVEMENTS
+  connect(
+      m_hotkey_scheduler, &HotkeyScheduler::OpenAchievements, this,
+      [this] {
+        Pause();
```

Comment:
Is there any particular reason why this pauses emulation? I'm not aware of any other hotkeys that work like this.